databricks/koalas/base.py

```diff
@@ -1202,7 +1202,8 @@ def isin(self, values) -> Union["Series", "Index"]:
                 " to isin(), you passed a [{values_type}]".format(values_type=type(values).__name__)
             )
 
-        return self._with_new_scol(self.spark.column.isin(list(values)))
+        values = values.tolist() if isinstance(values, np.ndarray) else list(values)
```

Comment:
```
>>> np.array([3, 4, 5]).tolist() == list(np.array([3, 4, 5]))
True
```
I'm wondering why we check `isinstance(values, np.ndarray)`?

Reply:
This is because the type of items is different.

```python
>>> for item in np.array([3, 4, 5]).tolist():
...     type(item)  # int
...
<class 'int'>
<class 'int'>
<class 'int'>

>>> for item in list(np.array([3, 4, 5])):
...     type(item)  # numpy.int64
...
<class 'numpy.int64'>
<class 'numpy.int64'>
<class 'numpy.int64'>
```

And, seems like PySpark's `isin` doesn't understand `numpy.int64`.

```python
>>> sdf.a.isin(np.int64)  # `sdf` is PySpark DataFrame
Traceback (most recent call last):
...
AttributeError: type object 'numpy.int64' has no attribute '_get_object_id'
```

Reply:
Thanks for the explanation! That's clear.